ansible/templates/killbill/killbill.properties.j2

```diff
@@ -132,9 +133,8 @@ org.killbill.dao.minIdle={{ lookup('env', 'KILLBILL_DAO_MIN_IDLE') }}
 {% if lookup('env', 'KILLBILL_DAO_MYSQL_SERVER_VERSION') %}
 org.killbill.dao.mysqlServerVersion={{ lookup('env', 'KILLBILL_DAO_MYSQL_SERVER_VERSION') }}
 {% endif %}
-{% if lookup('env', 'KILLBILL_DAO_PASSWORD') %}
+{# Allow for blank passwords (SSL certificates usecase) #}
 org.killbill.dao.password={{ lookup('env', 'KILLBILL_DAO_PASSWORD') }}
-{% endif %}
```

Comment:
If i understand correctly, before we would first check if such env variable exists before calling `lookup` and setting the value for `org.killbill.dao.password`; with the change, we now always set the value for `org.killbill.dao.password` based on the result of the `lookup` method which could then be empty and therefore allows us to have an empty pwd for SSL ?

Reply:
Yup - otherwise we default to the hardcoded value in config-magic (`killbill`).